src/sumBy.ts

```diff
@@ -11,6 +18,13 @@ import { purry } from "./purry";
  *      [{a: 5}, {a: 1}, {a: 3}],
  *      R.sumBy(x => x.a)
  *    ) // 9
+ *
+ *    R.pipe(
+ *      [{a: 5n}, {a: 1n}, {a: 3n}],
+ *      R.sumBy(x => x.a)
+ *    ) // 9n
+ *
+ *    R.pipe([] as {a: bigint}[], R.sumBy(x => x.a)) // 0
```

Comment:
This concept is much easier to understand when mentioned in the function's description instead of as an example. The example doesn't make it clear what it's trying to say. Check out how we did it in `sum`.

Reply:
Moved to the description for easier to understand